synapse_data_warehouse/database_roles/V2.33.0__database_analyst_roles.sql

```diff
@@ -0,0 +1,96 @@
+USE DATABASE {{ database_name }}; --noqa: JJ01,PRS,TMP
```

Comment:
Our initial state after #104 is merged.
![SNOW-200 (1)](https://github.com/user-attachments/assets/31ac5f0e-68ec-46d0-a77c-6336e472a106)